### DISCUSSION TITLE
Installation problem on 32-bit windows2008

### DISCUSSION THREAD

**Discussion by duansl** [ImageMagick/ImageMagick#3997] [Help]
hi, I downloaded these versions and tried to install them on my 32-bit operating system windows2008, but they all failed to install, saying "Not a valid win32 application". How can I install it successfully, or please tell me which version I need to install? thanks
![b6eb924f2110db6e52eecd44e434023](https://user-images.githubusercontent.com/24426108/127618890-cc0110fc-2853-4797-b6a7-ff2d9e0e4870.png)

**Comment by dlemstra** [comment]
It might be possible that for Windows 2008 you need a binary that has support for Windows XP. I see the same error message here: #2376.
